guide/additional-info/async-await.md

```diff
@@ -184,8 +184,8 @@ The return value of a `.reply()` with the `fetchReply` option set to `true` is a
 client.on('interactionCreate', async interaction => {
 	if (commandName === 'delete') {
 		try {
-			const replyMessage = await interaction.reply('This message will be deleted.', { fetchReply: true });
-			await replyMessage.delete({ timeout: 10000 });
+			await interaction.reply({ content: 'This message will be deleted.', fetchReply: true });
```

Comment:
```suggestion
const replyMessage = await interaction.reply({ content: 'This message will be deleted.', fetchReply: true });
```
`replyMessage` got undefined and the line below will error out.

Reply:
Missed that, thanks